Gemfile

```diff
@@ -7,37 +7,38 @@ ruby '2.7.2'
 gem 'rails', '~> 6.1.3', '>= 6.1.3.2'
 
 gem 'pg', '~> 1.1'
-# Use Puma as the app server
 gem 'puma', '~> 5.0'
-
-# Reduces boot times through caching; required in config/boot.rb
 gem 'bootsnap', '>= 1.4.4', require: false
-
-# Use Rack CORS for handling Cross-Origin Resource Sharing (CORS), making cross-origin AJAX possible
-# gem 'rack-cors'
-
 group :development, :test do
   gem 'byebug', platforms: [:mri, :mingw, :x64_mingw]
   gem "rspec-rails", "~> 5.0"
   gem "factory_bot_rails", "~> 6.2"
+<<<<<<< HEAD
```

Comment:
This looks like a merge conflict, this Gemfile will most likely not work.

Reply:
There are several instances of merge conflicts messages that were not removed after solving the conflicts, please remove all of then